### PR TITLE
chore: missing value-entities and views. Rest entities/services passing tests

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
Adding to the comments in unmanaged source files generated by codegen the kind of entity or service it is and its proto descriptor 
References #802

### CODE REVIEW

File: codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala [hunk 1]
```diff
@@ -43,13 +43,29 @@ object SourceGeneratorUtils {
...
+
+    val (kind, fileName) = service match {
+      case Left(serv: ModelBuilder.ActionService)      => ("Action Service", serv.fqn.parent.protoFileName)
+      case Left(serv: ModelBuilder.ViewService)        => ("View Service", serv.fqn.parent.protoFileName)
+      case Right(ent: ModelBuilder.EventSourcedEntity) => ("Event Sourced Entity Service", ent.fqn.parent.protoFileName)
+      case Right(ent: ModelBuilder.ValueEntity)        => ("Value Entity Service", ent.fqn.parent.protoFileName)
+      case Right(ent: ModelBuilder.ReplicatedEntity)   => ("Replicated Entity Service", ent.fqn.parent.protoFileName)
+    }
+    s"""// This class was initially generated based on the .proto definition by Akka Serverless tooling.
+       |// This is the implementation for the $kind described in your $fileName file.
```
Comment (johanandren) [inline codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:63]: include the relative proto path if possible, I guess we don't know if a user remapped `src/main/proto` in their project, but for `src/main/proto/com/example/my_service.proto` getting a `com/example/my_service.proto` would be nice

Comment (franciscolopezsancho) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:63]: I tried and didn't see the path in BuilderModel.Service or Entity. I'll have another look around to see what can use.

Comment (johanandren) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:63]: aren't you dropping the parent path just above when you pick up only the `protoFileName` from the fully qualified name? 

Perhaps @octonato knows from the top of his head given that he has been mucking about with the modelbuilder lately.

Comment (franciscolopezsancho) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:63]: It's all good. Thanks @johanandren

File: codegen/java-gen/src/test/resources/tests/action-service/named-new-style/generated-unmanaged/org/example/service/MyServiceNamedAction.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import com.google.protobuf.Empty;
 
 // This class was initially generated based on the .proto definition by Akka Serverless tooling.
+// This is the implementation for the Action Service described in your com/example/different/example-action.proto file.
```
Comment (franciscolopezsancho) [inline codegen/java-gen/src/test/resources/tests/action-service/named-new-style/generated-unmanaged/org/example/service/MyServiceNamedAction.java:9]: Here's the proof that it works as expected.

Comment (franciscolopezsancho) [reply codegen/java-gen/src/test/resources/tests/action-service/named-new-style/generated-unmanaged/org/example/service/MyServiceNamedAction.java:9]: the package defined in the service is `com.example.service` while the `.proto` is in `com/example/different`.

Comment (johanandren) [reply codegen/java-gen/src/test/resources/tests/action-service/named-new-style/generated-unmanaged/org/example/service/MyServiceNamedAction.java:9]: Nice!

**Review by johanandren — APPROVED** [review]
LGTM but would be good if @octonato could take a look as well before we merge

File: codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/generated-unmanaged/org/example/service/MyServiceAction.java [hunk 3]
```diff
@@ -6,6 +6,7 @@
 import org.external.ExternalDomain;
 
 // This class was initially generated based on the .proto definition by Akka Serverless tooling.
+// This is the implementation for the Action Service described in your example-action.proto file.
```
Comment (octonato) [inline codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/generated-unmanaged/org/example/service/MyServiceAction.java:9]: Full path to proto file now showing here and below? Something missing?

File: codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala [hunk 4]
```diff
@@ -43,13 +43,29 @@ object SourceGeneratorUtils {
...
        |// or delete it so it is regenerated as needed.""".stripMargin
 
+  def unmanagedComment(service: Either[ModelBuilder.Service, ModelBuilder.Entity]) = {
+
+    val (kind, fileName) = service match {
+      case Left(serv: ModelBuilder.ActionService)      => ("Action Service", serv.fqn.parent.protoFileName)
+      case Left(serv: ModelBuilder.ViewService)        => ("View Service", serv.fqn.parent.protoFileName)
+      case Right(ent: ModelBuilder.EventSourcedEntity) => ("Event Sourced Entity Service", ent.fqn.parent.protoFileName)
+      case Right(ent: ModelBuilder.ValueEntity)        => ("Value Entity Service", ent.fqn.parent.protoFileName)
+      case Right(ent: ModelBuilder.ReplicatedEntity)   => ("Replicated Entity Service", ent.fqn.parent.protoFileName)
```
Comment (octonato) [inline codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:60]: Either is too often used to express computations that may fail and very rarely to express two possible valid out comes. Nice to see it being used that way. 👍

**Review by octonato — APPROVED** [review]
LGTM

But I think that there is something missing. Only one generated file is showing the full path to the proto file.

File: codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/generated-unmanaged/org/example/service/MyServiceAction.java (hunk 3 above)
Comment (franciscolopezsancho) [reply codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/generated-unmanaged/org/example/service/MyServiceAction.java:9]: this example, as all the rest but one, is in the base src/main/proto folder. Only the path underneath, if any, appears. 
Does this make sense?

File: codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala (hunk 4 above)
Comment (franciscolopezsancho) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala:60]: :) Thanks!

File: codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/generated-unmanaged/org/example/service/MyServiceAction.java (hunk 3 above)
Comment (franciscolopezsancho) [reply codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/generated-unmanaged/org/example/service/MyServiceAction.java:9]: have a look at here please
https://github.com/lightbend/akkaserverless-java-sdk/pull/820/files/315e6d01ea0c4a95ef7bcfc04626296d63fbf0c1#diff-2e0727ef0ef5a9a962fe8366da4c1ffe4b37e76ef5f6303f93898f66beda3cb7

**Pull request merged**